.github/workflows/pr_info_check.yml

```diff
@@ -0,0 +1,83 @@
+# NM was here
+
+name: 'PR Info Check'
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check PR JIRA reference
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(\[WIP\] )?(D4CHAT-\d+).*$'
+          error: 'Pull Request name must start with a JIRA ticket reference'
+          flags: 's'
+          excludeDescription: 'true'
+          excludeTitle: 'false'
+          checkAllCommitMessages: 'false'
+      - name: Check PR description
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(\[WIP\] )?([A-Z0-9]+-\d+) .{10,}$'
+          error: 'Pull Request name must have meaningful description'
+          flags: 's'
+          excludeDescription: 'true'
+          excludeTitle: 'false'
+          checkAllCommitMessages: 'false'
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(feat|fix|docs|style|refactor|test|chore): .*$'
```

Comment:
Do you mind to discuss these keywords in the next meeting?

Reply:
Sure, hope for it)

Reply:
Discussed